Data/CorrectionTable.cpp

```diff
@@ -16,7 +16,7 @@ void CorrectionTable::Print() const {
   if (sys.size() == 0 && antiSys.size() == 0)
     return; // don't print if empty
   std::stringstream out;
-  ComPWA::TableFormater table(&out);
+  ComPWA::Tools::TableFormatter table(&out);
   table.addColumn("Bin", 12);             // add empty first column
```

Comment:
`TableFormatter` is located in `Core` but is in namespace `Tools`. That could be a bit more consistent.